plugins/performance-lab/load.php

```diff
@@ -145,8 +145,9 @@ function perflab_get_standalone_plugin_version_constants(): array {
  * the frontend.
  *
  * This function will short-circuit if at least one of the constants
- * 'PERFLAB_DISABLE_SERVER_TIMING' or 'PERFLAB_DISABLE_OBJECT_CACHE_DROPIN' is
- * set as true.
+ * 'PERFLAB_DISABLE_SERVER_TIMING' or
+ * 'PERFLAB_DISABLE_OBJECT_CACHE_DROPIN' is set as true or if the
+ * 'PERFLAB_PLACE_OBJECT_CACHE_DROPIN' constant is not set to a truthy value.
  *
  * @since 1.8.0
  * @since 2.1.0 No longer attempts to use two of the drop-ins together.
```

Comment:
Just a note: Could we include the changes in the `@since` annotation so users can easily see when they were introduced?

Reply:
```suggestion
 * @since 2.1.0 No longer attempts to use two of the drop-ins together.
 * @since n.e.x.t No longer places the drop-in on new sites by default, unless the `PERFLAB_PLACE_OBJECT_CACHE_DROPIN` constant is set to true.
```